flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/AmbiguousTableFactoryException.java

```diff
@@ -26,6 +27,7 @@
 import java.util.stream.Collectors;
 
 /** Exception for finding more than one {@link TableFactory} for the given properties. */
+@PublicEvolving
 public class AmbiguousTableFactoryException extends RuntimeException {
```

Comment:
Same for `NoMatchingTableFactoryException`?

Reply:
Indeed!